### ISSUE TITLE
No bounty instantDummyCaster

### ISSUE THREAD

**Issue by godbit** [Lapsang-boys/Rex-Temporis#51]
I think i have figured out why we get no bounty from spells cast by instantDummy caster. It is because once the spell has been cast the owner changes so out player is no longer entitled to the bounty.
```wurst
private static function finish()
   caster.setOwner(DUMMY_PLAYER, false)
   caster.removeAbility(id)
   caster.setPos(boundMax)
```
This is probably intended in the standard library but we need to fix this in some way.

**Comment by karlek** [comment]
Ah, yes smart! Btw, nice palette of labels.

**Comment by karlek** [comment]
What about, asking making a pull request for `wurstSdlib2` with player owned dummies? If they have locust, you can't select them anyway so should be okay.

**Comment by godbit** [comment]
Yeah that could be a ting, for instance that you call InstantDummyCaster with a flag meaning that it will be kept for that owner.

Another one could be to add a time in the call which is the time the dummy will be kept without being released. This will allow you to cast a DoT that lasts for 10 sec and the release after say 12 sec.
